src/heitech.zer0mqXt.core/Patterns/SendReceive/Receiver.cs

```diff
@@ -19,7 +19,7 @@ internal Receiver(SocketConfiguration configuration)
         public XtResult SetupReceiver<TMessage>(Action<TMessage> callback, CancellationToken token = default)
             where TMessage : class, new()
         {
-            var setupResult = _responder.Respond<TMessage, EmptyResponse>((msg) => { callback(msg); return new EmptyResponse(); }, token);
+            var setupResult = _responder.Respond<TMessage, EmptyResponse>((msg) => { callback(msg); return new EmptyResponse(); }, () => new EmptyResponse(), token);
```

Comment:
Perhaps the `EmptyResponse` could be a singleton ... or at least have a `static readonly Instance` property to avoid additional allocations of this stateless object 🤔.

Reply:
ok, that is optimization in 'ns' area, which is certainly doable but probably of no concern for this type of library for now...
Will still see what I can do in this PR. Should not require too much change though.